connectors/migrations/db/migration_79.sql

```diff
@@ -0,0 +1,2 @@
+-- Migration created on Jun 19, 2025
+ALTER TABLE "public"."webcrawler_configurations" ADD COLUMN "crawlId" VARCHAR(36) DEFAULT NULL;
```

Comment:
To respect Dust's rule 🙃 
```suggestion
ALTER TABLE "public"."webcrawler_configurations" ADD COLUMN "crawlId" VARCHAR(64) DEFAULT NULL;
```